packages/mariadb/test/env.coffee

```diff
@@ -0,0 +1,97 @@
+
+path = require 'path'
+nikita = require 'nikita'
+{env, tags} = require './test'
+{cluster} = require '../env'
+
+
+describe 'Test environment', ->
+
+  @timeout 1500000 # Extended timeout for these lengthy test
+
+  describe 'Cluster status', ->
+
+    it 'Up and running', ->
```

Comment:
here, you are testing the "cluster" framework and not mariadb. it is fine for now but it shall be moved somewhere else at some point and not be associated with mariadb